intro-to-R-tidyverse/exercise_03a-intro_to_tidyverse.Rmd

```diff
@@ -241,7 +241,7 @@ Add a layer with `geom_density()` to make a density plot of the medians.
 
 ```
 
-If you haven't, don't forget to add `xlab()` and `ylab()` to your plot so your 
+If you haven't, don't forget to add x and y labels with `labs()` to your plot so your 
```

Comment:
See above: this change will need to be made in https://github.com/AlexsLemonade/exercise-notebook-answers/blob/master/intro-to-R-tidyverse/exercise_03a-intro_to_tidyverse-solved.Rmd